code/style.css

```diff
@@ -1,4 +1,193 @@
+  font-family: 'Arial Narrow' , sans-serif;
+}
+
+/*header*/
+header {
+  display: flex;
+  padding-bottom: 1%;
+}
+header img {
+  height: 8vw;
+  padding: 1% 2% 0 0;
+}
+.site-name {
+  align-self: center;
+  font-size: 3vw;
+}
+nav {
+  display: none;
+ }
+.hamburger {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 1% 1% 0 0;
+}
+.site-name:hover {
+color: #D3DEDC;
+}
+
+/*big news section*/
+.bignews-container {
+  display: box;
+  display: flex;
+  box-align: center;
+  align-items: center;
+  box-pack: center;
+  justify-content: center;
+  text-align: center;
+  margin-bottom: 3vw;
+}
+.bignews-image {
+  position: relative;
+  width: 100%;
+}
+.bignews-text {
+  color: #FFBD35;
+  font-weight: 1500;
+  position: absolute;
+  text-shadow: 1vw 1vw 2vw black;
+  -webkit-text-stroke: 0.05vw black;
+}
+.bignews-text h1 {
+  font-size: 8vw;
+  margin-bottom: 0;
+}
+.bignews-text p {
+  font-size: 2vw;
+  padding: 3vw;
+  /* text-shadow: 1vw 1vw 2vw black; */
+}
+.bignews-image:hover {
+  -webkit-transform: scaleX(-1);
+  transform: scaleX(-1);
+}
+
+/*small news section*/
+.smallnews-container {
+  background-color: #D3DEDC;
+  padding: 5vw 0 5vw 0;
+}
+.smallnews-items {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  font-size: 1.5vw;
+  gap: 4vw;
+}
+.smallnews-container img {
+  height: 55vw;
+  width: 55vw;
+}
+.smallnews-items h1 {
+   font-weight: 600;
+   padding: 0 2vw 0 2vw;
+   text-align: center;
+}
+ .scandic {
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .sixt {
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .leisure {
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .virgin {
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .mandarin {
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .hertz{
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .greece {
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .ethiad {
+  box-shadow: 2vw 1vw 1vw black;
+  background-color: whitesmoke;
+ }
+ .smallnews-container img:hover {
+  -webkit-transform: scaleX(-1);
+  transform: scaleX(-1);
+}
+
+ /*footer*/
+ .footer-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+ }
+ .footer-icons {
+   font-size: 3vw;
+ }
+ .footer-icons:hover {
+  color: #D3DEDC;
+ }
+ .copywrite {
+   font-size: 2vw;
+   font-weight: lighter;
+   padding-left: 10vw;
+ }
+
+ /*media queries*/
+ @media screen and (min-width: 668px) {
+  .hamburger {
+    display: none;
+  }
+  nav {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: flex-end;
+    font-size: 2vw;
+    padding-right: 3vw;
+  }
+  a {
+    padding: 1vw;
+    color: black;
+    text-decoration: none;
+  }
+  a:hover {
+    color: #D3DEDC;
+  }
+  a:-webkit-any-link {
+    cursor: pointer;
```

Comment:
I think you can change the cursor under a:hover too and it will have the same effect, something like:
```suggestion
  a:hover {
    color: #D3DEDC;
    cursor: pointer;
    }
```